src/phrases/phrases-en.js

```diff
@@ -426,10 +435,16 @@ const phrases = {
       content: 'Content',
       notes: 'Notes',
       metaDescription: 'Meta description',
+      metaImageAlt: 'Alt-text',
+      alttext: 'Alt-text',
```

Comment:
Hvorfor to felt med samme oversettelse?

Reply:
Hmm, to forskjellige felt med litt forskjellige betydning. Skjønner hva du tenker på.